modules/jupyter-widget/src/playground/utils/google-maps-utils.js

```diff
@@ -0,0 +1,31 @@
+/* global window */
+import * as deck from '../../deck-bundle';
+
+export function createGoogleMapsDeckOverlay({
+  props,
+  googleMapsKey,
+  onClick,
+  container,
+  getTooltip,
+  onComplete
+}) {
+  googleMapsKey = props.googleMapsKey || googleMapsKey;
+  if (!googleMapsKey) {
+    deck.log.warn('No Google Maps API key set');
```

Comment:
This returns a function. You need to call it: `log.warn(...)()`

Make sure that you test this use case.